src/components/filterbox.jsx

```diff
@@ -101,10 +101,10 @@ function useSelections(model, layout, selfRef) {
   const onRowClick = async ({ rowData }) => {
     if (rowData) {
       const rowDataToModify = rowData;
-      if (rowData[0].qState !== 'S') {
+      if (rowData[0].qState !== 'S' && rowData[0].qState !== 'XS') {
```

Comment:
I'm not sure this case is 100% correct, might be a scenario where mapping it to 'S' is wrong, but I cannot test now and edge cases here are fine by me.